src/gui/optionsdialog.h

```diff
@@ -107,6 +107,14 @@ private slots:
     void on_registerDNSBtn_clicked();
     void setLocale(const QString &localeStr);
 
+    void on_checkBoxAltPauseUploads_toggled(bool checked);
+
+    void on_checkBoxAltPauseDownloads_toggled(bool checked);
+
+    void on_checkUploadLimitAlt_toggled(bool checked);
+
```

Comment:
Why the empty lines between these? Looks like they belong together.

Reply:
Looks like generated this way. I will remove these lines.